test-apps/display-test-app/src/common/DtaConfiguration.ts

```diff
@@ -55,6 +54,7 @@ export interface DtaStringConfiguration {
   oidcClientId?: string; // default is undefined, used for auth setup
   oidcScope?: string; // default is undefined, used for auth setup
   oidcRedirectUri?: string; // default is undefined, used for auth setup
+  useFrontendTiles?: string; // if set, specifies url for @itwin/frontend-tiles to obtain tile trees for spatial views.  See README.md
```

Comment:
Rename to `frontendTilesUrlTemplate` or something similarly descriptive.

Reply:
Done